### DISCUSSION TITLE
Option "CDK" is missing in the AWS Toolkit under Developer Tools

### DISCUSSION THREAD

**Discussion by Fazzke** [aws/aws-cdk#27714] [General]
When I lookup tutorials or documentations, there is always an option "CDK" under the Developer Tools of the AWS Toolkit. However - in my Toolkit only the options "CodeWhisperer" and "CodeCatalyst" are visible. 

Do you know what I might be doing wrong which causes this issue? I'm using a JetBrains IDE.

**Comment by indrora** [comment]
Could you be a little more specific? Screenshots, perhaps?
